pkgs/tools/networking/network-manager/fix-paths.patch

```diff
@@ -61,29 +61,42 @@ index 51ca46d2b..0c04cc216 100644
      depends: libnm_gir,
    )
  endif
+diff --git a/meson.build b/meson.build
+index a2d925a7e..23c85f128 100644
+--- a/meson.build
++++ b/meson.build
+@@ -522,7 +522,7 @@ polkit_agent_dep = dependency('polkit-agent-1', version: '>= 0.97', required : f
+ if polkit_agent_dep.found()
+ 	config_h.set_quoted('POLKIT_PACKAGE_PREFIX', polkit_agent_dep.get_pkgconfig_variable('prefix'))
+ else
+-	config_h.set_quoted('POLKIT_PACKAGE_PREFIX', '/usr')
++	config_h.set_quoted('POLKIT_PACKAGE_PREFIX', '@polkit@')
```

Comment:
Actually, looking at the use case, this is not even correct.

We need to patch https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/75bc21c4cf2c54e368d665302760cfe41b1b6814/clients/common/nm-polkit-listener.c#L555 to `/run/wrappers/bin/polkit-agent-helper-1`

Reply:
Exactly, will fix.

Reply:
I've hardcoded it for now by not using `POLKIT_PACKGE_PREFIX` as suggested.
How about upstream they can just have
```
option('polkit_package_prefix', type: 'string', value: '/usr', description: 'Prefix for the polkit package')
```

and just use that option value in the `else`. That way we could just put the path in `mesonFlags`?

Reply:
An option for adjusting just the prefix would not help us much, we can already do as such using `PKG_CONFIG_POLKIT_AGENT_1_PREFIX` environment variable. The only part we share with the upstream path is the executable name:

```diff
-/usr/lib/polkit-1/polkit-agent-helper-1
+/run/wrappers/bin/polkit-agent-helper-1
```